pootle/settings/50-project.conf

```diff
@@ -38,9 +38,7 @@ MIDDLEWARE_CLASSES = [
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [
-            working_path(os.path.join('templates')),
-        ],
+        'DIRS': [],
```

Comment:
Just tested and it seems that `APP_DIRS` conflicts with having loaders defined. If we have no filesystem loader then we won't be able to allow customization.